training/distributed_training/pytorch/data_parallel/yolov5/yolov5.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        "aws ecr get-login-password --region us-east-1 docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com",
</pre><p>Can you use the user's current region rather than hard-coding us-east-1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>